airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/SqlOperationsUtils.kt

```diff
@@ -17,136 +16,90 @@ import java.util.*
 import java.util.function.Consumer
 import java.util.function.Supplier
 
-object SqlOperationsUtils {
-    /**
-     * Inserts "raw" records in a single query. The purpose of helper to abstract away
-     * database-specific SQL syntax from this query.
-     *
-     * @param insertQueryComponent the first line of the query e.g. INSERT INTO public.users (ab_id,
-     * data, emitted_at)
-     * @param recordQueryComponent query template for a full record e.g. (?, ?::jsonb ?)
-     * @param jdbcDatabase jdbc database
-     * @param records records to write
-     * @throws SQLException exception
-     */
-    @JvmStatic
-    @Throws(SQLException::class)
-    fun insertRawRecordsInSingleQuery(
-        insertQueryComponent: String?,
-        recordQueryComponent: String?,
-        jdbcDatabase: JdbcDatabase,
-        records: List<PartialAirbyteMessage>
-    ) {
-        insertRawRecordsInSingleQuery(
-            insertQueryComponent,
-            recordQueryComponent,
-            jdbcDatabase,
-            records,
-            { UUID.randomUUID() },
-            true
-        )
+/**
+ * Inserts "raw" records in a single query. The purpose of helper to abstract away database-specific
+ * SQL syntax from this query.
+ *
+ * @param insertQueryComponent the first line of the query e.g. INSERT INTO public.users (ab_id,
+ * data, emitted_at)
+ * @param recordQueryComponent query template for a full record e.g. (?, ?::jsonb ?)
+ * @param jdbcDatabase jdbc database
+ * @param records records to write
+ * @throws SQLException exception
+ */
+@Throws(SQLException::class)
```

Comment:
This is still used in RedshiftSqlOperations v1 insert query call (until everyone adopts the DV2 breaking change that code could get invoked). So keep the `@JvmStatic`

Reply:
Its still static just moved outside the object since it was the only method in the object

Reply:
interesting, so kotlin doesn't really have static is what i read. In java ctx is it still invocable as `SqlOpUtils.insert..` ?

Reply:
Yeah in the stacked PR on top I reference this method via 
```
import static io.airbyte.cdk.integrations.destination.jdbc.SqlOperationsUtilsKt.insertRawRecordsInSingleQuery
```